assets/blocks/email-editor.js

```diff
@@ -22,24 +22,42 @@ export default function handleEmailBlocksEditor() {
 	 * @param {Object} settings Block settings.
 	 */
 	function removeIrrelevantSettings( settings ) {
+		const supports = { ...( settings.supports ? settings.supports : {} ) };
+
+		// Remove font family setting.
 		if (
 			has( settings, 'supports.typography.fontFamily' ) ||
 			has( settings, 'supports.typography.__experimentalFontFamily' )
 		) {
-			settings = {
-				...settings,
-				supports: {
-					...settings.supports,
-					typography: {
-						...settings.supports.typography,
-						__experimentalFontFamily: false,
-						fontFamily: false,
-					},
-				},
+			supports.typography = {
+				...supports.typography,
+				__experimentalFontFamily: false,
+				fontFamily: false,
 			};
 		}
 
-		return settings;
+		// Remove alignWide setting.
+		if (
```

Comment:
I think you can replace this code block with lodash just with:
`has( settings, 'supports.alignWide' , false)`

Reply:
Just checked the doc, seems like `has` takes [two arguments](https://lodash.com/docs/#has), am I missing something? Code editor is also throwing an invalid parameter error

Reply:
Removed the second check here https://github.com/Automattic/sensei/pull/6558/commits/41b37cc9973bc0fb158d294a69cb1904443cea54, probably we don't need that

Reply:
> Just checked the doc, seems like `has` takes [two arguments](https://lodash.com/docs/#has), am I missing something? Code editor is also throwing an invalid parameter error

Sorry, I was trying to say 'set' because it set safely the value.

```
set( settings, 'supports.alignWide' , false)
```

but it is just a small improvement and not required.